src/games/strategy/triplea/ui/MapPanel.java

```diff
@@ -128,25 +126,25 @@ public void run() {
         });
       }
     });
-    recreateTiles(data, m_uiContext);
-    m_uiContext.addActive(new Active() {
+    recreateTiles(data, this.uiContext);
```

Comment:
prefer to avoid using 'this' when it is not necessary.  The rest of the project is relatively consistent about that. Small point.